src/sentry/api/endpoints/project_details.py

```diff
@@ -353,6 +354,10 @@ def put(self, request, project):
         if result.get('storeCrashReports') is not None:
             if project.update_option('sentry:store_crash_reports', result['storeCrashReports']):
                 changed_proj_settings['sentry:store_crash_reports'] = result['storeCrashReports']
+        if result.get('relayPiiConfig') is not None:
+            if project.update_option('sentry:relay_pii_config', result['relayPiiConfig']):
+                changed_proj_settings['sentry:relay_pii_config'] = result['relayPiiConfig'].strip(
+                ) or None
```

Comment:
Should we do any validation on the structure of the JSON blob they are giving us?